x-pack/plugins/ml/public/timeseriesexplorer/components/timeseries_chart/timeseries_chart.js

```diff
@@ -33,14 +36,24 @@ import {
   numTicksForDateFormat,
   showMultiBucketAnomalyMarker,
   showMultiBucketAnomalyTooltip,
-} from '../util/chart_utils';
+} from '../../../util/chart_utils';
 import { TimeBuckets } from 'ui/time_buckets';
-import { mlAnomaliesTableService } from '../components/anomalies_table/anomalies_table_service';
-import ContextChartMask from './context_chart_mask';
-import { findChartPointForAnomalyTime } from './timeseriesexplorer_utils';
-import { mlEscape } from '../util/string_utils';
-import { mlFieldFormatService } from '../services/field_format_service';
-import { mlChartTooltipService } from '../components/chart_tooltip/chart_tooltip_service';
+import { mlAnomaliesTableService } from '../../../components/anomalies_table/anomalies_table_service';
+import { ContextChartMask } from '../context_chart_mask';
+import { findChartPointForAnomalyTime } from '../../timeseriesexplorer_utils';
+import { mlEscape } from '../../../util/string_utils';
+import { mlFieldFormatService } from '../../../services/field_format_service';
+import { mlChartTooltipService } from '../../../components/chart_tooltip/chart_tooltip_service';
+import {
+  getAnnotationBrush,
+  getAnnotationLevels,
+  renderAnnotations,
+
```

Comment:
Nit - extra space.

Reply:
This was actually by intention :) It's not enforced via linting in the `js` files, but in TypeScript files, `prettier` forces you to sort alphabetically, but you can build separate groups by splitting with a line. Was just trying to get used to that :).